js/hangman.js

```diff
@@ -0,0 +1,129 @@
+let pass = "Bez pracy nie ma kołaczy";
+pass = pass.toUpperCase();
+
+let passHidden = "";
+
+let passLen = pass.length;
+
+let missedCount = 0;
+
+let yes = new Audio("samples/yes.wav");
+let no = new Audio("samples/no.wav");
+
+for (i=0; i<passLen; i++)
+{
+    if (pass.charAt(i) == " ") passHidden = passHidden + " ";
+    else passHidden = passHidden + "-";
+}
+
+function writePass()
+{
+    document.getElementById("label").innerHTML = passHidden;
+}
+
+window.onload = start;
+
+let letters = new Array(35);
```

Comment:
Reformat code, avoid adding letters by accessing single Array element, you can simplify this part